improvelib/utils.py

```diff
@@ -39,6 +39,17 @@ def str2bool(v: str) -> bool:
         raise argparse.ArgumentTypeError("Boolean value expected.")
 
 
+def cast_value(s):
+    """Cast to numeric if possbile"""
+    try:
+        return int(s)
+    except ValueError:
+        try:
+            return float(s)
+        except ValueError:
```

Comment:
@wilke Alex added this because the numeric parameters were being read in as strings. I observed that as well when putting together the unit tests.

Reply:
This is a quick fix. We can look at this later if we want to make this change elsewhere.

Reply:
@jonesse3 Yes, but not necessary. 

https://docs.python.org/3/library/configparser.html

Apart from [getboolean()](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getboolean), config parsers also provide equivalent [getint()](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getint) and [getfloat()](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getfloat) methods. You can register your own converters and customize the provided ones. [[1]](https://docs.python.org/3/library/configparser.html#id15)